schema/mysql/v8/visibility/versioned/v1.9/add_version_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE executions_visibility ADD COLUMN _version BIGINT NOT NULL DEFAULT 0;
+ALTER TABLE custom_search_attributes ADD COLUMN _version BIGINT NOT NULL DEFAULT 0;
```

Comment:
The "⛔" is because it doesn't have a final line break ("blank line" after line 2. It's kind of a standard for the last line of a file to be a blank line.

Reply:
Thanks, I've never noticed that symbol.